include/cantera/base/NoExitLogger.h

```diff
@@ -0,0 +1,26 @@
+#ifndef NOEXITLOGGER_H
+#define NOEXITLOGGER_H
+
+#include <string>
+#include "cantera/base/logger.h"
+
+using namespace std;
```

Comment:
`using namespace std` should not appear in a header file (and doesn't appear to be necessary here in any case).